tenets/codelingo/go/use-ticker-in-for-switch/expected.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+   "Comment": "Make one ticker as opposed to making a new timer each iteration.",
```

Comment:
The user has no idea what "ticker" is.

Reply:
"Create a ticker outside of the `for` loop with `time.NewTicker(<time period>)` and use it here to avoid creating a new object in memory on each iteration of the loop."